src/redirects.njk

```diff
@@ -0,0 +1,75 @@
+---
+permalink: /_redirects

```

Comment:
I'm confused why this file exists, and the file without this extension?

Reply:
It’s not ready yet; the review was requested automatically.

Reply:
I think it already works on my machine?

Reply:
Keep this file, remove the changes in .eleventy.js and it's ready?